14/umbraco-cms/fundamentals/data/data-types/README.md

```diff
@@ -8,7 +8,7 @@ _A Data Type defines the type of input for a property. So when adding a property
 
 ## What is a Data Type?
 
-A Data Type can be something basic (textstring, number, true/false,...) or it can be more complex (multi node tree picker, image cropper, Grid Layout).
+A Data Type can be something basic (TextString, Number, True/False and so on) or it can be more complex (Multi Node Tree Picker, Image Cropper, Block Grid and so on).
```

Comment:
```suggestion
A Data Type can be something basic such as TextString, Number, True/False and so on. Or it can be more complex such as Multi Node Tree Picker, Image Cropper, Block Grid and so on.
```